### ISSUE TITLE
Certificate Error

### ISSUE THREAD

**Issue by gjsduarte** [hashicorp/vscode-terraform#535]
The extension isn't starting anymore, and the output on the Developer Tools console is:

![image](https://user-images.githubusercontent.com/288812/103694538-035ac800-4f93-11eb-9f79-96869ae0bbb6.png)

Any ideas?

**Comment by radeksimko** [comment]
Hi @gjsduarte 
Can you confirm you are able to install any other extension and this occurs only when installing the Terraform extension?

I wasn't able to reproduce this.

**Comment by gjsduarte** [comment]
Hey @radeksimko
Yes, I have a whole bunch of extensions installed, and they all work correctly. 
BTW, the error happens when loading the extension, not at install time.

**Comment by radeksimko** [comment]
The only hostname which the extension talks to on activation is `releases.hashicorp.com` which has an up-to-date certificate. Is it possible that you are in a network with some kind of proxy, which is presenting a different (invalid) certificate to you?

Can you try opening https://releases.hashicorp.com in your browser or `curl` it from your terminal?

**Comment by gjsduarte** [comment]
I found the problem. It seems I had some weird expired certificate installed on my machine causing the issue.
I'll leave the steps I took to debug the problem as a reference:
 - I switched `http.systemCertificates` to `false` in my `settings.json` and the extension immediately started working. With this, I was able to confirm that the problem was most likely on my machine.
 - I scanned all my installed certificates for expired ones and deleted them (using https://gist.github.com/gjsduarte/dab9e49f9f3c85e6ff6de7db34d81285)
 - I switched `http.systemCertificates` back to `true` and reloaded my vscode, and the extension was still working. Problem solved
 
Thanks for your support @radeksimko!
